src/SFE.scss

```diff
@@ -12,7 +12,7 @@
   font-weight: inherit;
   line-height: inherit;
   color: inherit;
-  text-align: left;
+  text-align: inherit;
```

Comment:
Farhanah and I weren't sure why this was set to left, so we decided to change it to `inherit`.